### DISCUSSION TITLE
Embedded RavenDB for unit tests using node.js & MacOS with Apple silicon

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#14056] [General]
Hi,

I am setting up a new project in node.js using latest version of RavenDB. Most our development team are using new Macbook pros with Apple silicon.

I want to build a test framework for integration tests but given there is no download for RavenDB for Apple silicon I am wondering how you would suggest we approach this. 

Is our only option to run RavenDB in a docker container, are you planning on releasing an Apple compatible build anytime soon?

Thanks

**Comment by haludi** [comment]
Hi,

The current option is to use docker.
We have an open issue that you can follow -
https://issues.hibernatingrhinos.com/issue/RavenDB-18446
But we do not have a deadline.

Best regards,
Igal
